File: AREA OF SHAPE.cs
using System;

class Program
{
    static void Main()
    {
        Console.WriteLine("Choose a shape to calculate its area:");
        Console.WriteLine("1. Circle");
        Console.WriteLine("2. Rectangle");
        Console.WriteLine("3. Triangle");
        
        int choice = int.Parse(Console.ReadLine());
        
        switch (choice)
        {
            case 1:
                CalculateCircleArea();
                break;
            case 2:
                CalculateRectangleArea();
                break;
            case 3:
                CalculateTriangleArea();
                break;
            default:
                Console.WriteLine("Invalid choice. Please choose 1, 2, or 3.");
                break;
        }
    }

    static void CalculateCircleArea()
    {
        Console.Write("Enter the radius of the circle: ");
        double radius = double.Parse(Console.ReadLine());
        double area = Math.PI * Math.Pow(radius, 2);
        Console.WriteLine("The area of the circle is: " + area);
    }

    static void CalculateRectangleArea()
    {
        Console.Write("Enter the length of the rectangle: ");
        double length = double.Parse(Console.ReadLine());
        Console.Write("Enter the width of the rectangle: ");
        double width = double.Parse(Console.ReadLine());
        double area = length * width;
        Console.WriteLine("The area of the rectangle is: " + area);
    }

    static void CalculateTriangleArea()
    {
        Console.Write("Enter the base of the triangle: ");
        double baseLength = double.Parse(Console.ReadLine());
        Console.Write("Enter the height of the triangle: ");
        double height = double.Parse(Console.ReadLine());
        double area = 0.5 * baseLength * height;
        Console.WriteLine("The area of the triangle is: " + area);
    }
}

File: AREA OF SHAPE.py
def pi():
    return 3.142

def circle():
    radius = float(input("Enter the radius of the circle: "))
    area = pi() * radius ** 2
    return area

def square():
    side = float(input("Enter the side length of the square: "))
    area = side ** 2
    return area

def rectangle():
    length = float(input("Enter the length of the rectangle: "))
    width = float(input("Enter the width of the rectangle: "))
    area = length * width
    return area

def triangle():
    base = float(input("Enter the base of the triangle: "))
    height = float(input("Enter the height of the triangle: "))
    area = 0.5 * base * height
    return area

def calculate_area():
    print("Choose a shape - Circle, Square, Rectangle, or Triangle: ")
    shape = input().lower()
    area = None  # Initialize area

    if shape == "circle":
        area = circle()
        
    elif shape == "square":
        area = square()
        
    elif shape == "rectangle":
        area = rectangle()
        
    elif shape == "triangle":
        area = triangle()

    else:
        print("Invalid shape")
        return

    print(f"The area of the {shape} is: {area}")

#call the function to run the program
calculate_area()

